### PR TITLE
[Mac] Check for mono 5.10.124, or 5.8.0.130 to signal a mono which ca…

### PR DESCRIPTION
**Pull request by Therzok**
…n't handle csc compiled xammac

### CODE REVIEW

File: main/build/MacOSX/monostub.mm [hunk 1]
```diff
@@ -429,9 +431,17 @@ int main (int argc, char **argv)
...
+	// into the broken range of [5.9.0 5.10.0.123]
 	if (!check_mono_version (mono_version, [req_mono_version UTF8String])) {
-		NSString *msg = [NSString stringWithFormat:@"This application requires a newer version (%s+) of the Mono framework.", [req_mono_version UTF8String]];
-		exit_with_message ((char *)[msg UTF8String], argv[0]);
+		if (check_mono_version (mono_version, [req_mono_version_stable UTF8String])) {
+			if (check_mono_version (mono_version, "5.9.0")) {
+				NSString *msg = [NSString stringWithFormat:@"This application requires a newer version (%s+) of the Mono framework.", [req_mono_version UTF8String]];
+				exit_with_message ((char *)[msg UTF8String], argv[0]);
+			}
+		}
```
Comment (alanmcgovern) [inline main/build/MacOSX/monostub.mm]: ❤️

**Comment by Therzok** [comment]
I merged the wrong PR :/

**Comment by alanmcgovern** [comment]
it's ok! I'd have merged it anyway :)

**Comment by alanmcgovern** [comment]
I was just writing a message to say "What about master?" :P

**Pull request merged**